tools/pythonpkg/src/native/python_conversion.cpp

```diff
@@ -210,6 +210,29 @@ Value TransformListValue(py::handle ele, const LogicalType &target_type = Logica
 	return Value::LIST(element_type, values);
 }
 
+Value TransformArrayValue(py::handle ele, const LogicalType &target_type = LogicalType::UNKNOWN) {
```

Comment:
The way you can test this is by using `duckdb.execute("query with prepared parameters", {"param_name": param})`

The `target_type` is tested by creating `duckdb.Value(obj, <type object>)`
When scanning the `duckdb.Value` object we unwrap it and pass the enclosed type object as `target_type`